### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by thorium-zero**

### CODE REVIEW

File: db/models.py [hunk 1]
```diff
@@ -14,3 +14,35 @@ class Actor(models.Model):
...
+        return self.rows * self.seats_in_row
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey("CinemaHall", on_delete=models.CASCADE)
+    movie = models.ForeignKey(Movie, on_delete=models.CASCADE)
+
+    def __str__(self) -> str:
+        return f"{self.movie} {self.show_time.strftime('%Y-%m-%d %H:%M:%S')}"
```
Comment (YuriiKindrat) [inline db/models.py:48]: show time will be saved at the same format, so there is no need to use strftime )

File: services/cinema_hall.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> list[CinemaHall]:
+    return CinemaHall.objects.all()
```
Comment (YuriiKindrat) [inline services/cinema_hall.py:5]: are u sure about list type annotation ? 
it seems to me, that method all returns queryset)

File: services/movie.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+from db.models import Movie
+from django.db.models import Q
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> list[Movie]:
```
Comment (YuriiKindrat) [inline services/movie.py:8]: same is here)

**Review by YuriiKindrat — CHANGES_REQUESTED** [review]

File: services/cinema_hall.py (hunk 2 above)
Comment (thorium-zero) [reply services/cinema_hall.py:5]: ffs, I knew I'm going to forget to change this :(

File: services/movie_session.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+from db.models import CinemaHall, Movie, MovieSession
+
+
+def create_movie_session(
+    movie_show_time: datetime,
+    movie_id: int,
+    cinema_hall_id: int
+) -> MovieSession | None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
```
[the remark this answers is not in the document]
Comment (thorium-zero) [reply services/movie_session.py:12]: yeah, I was going to add some `nullability` checks here, so I could validate incoming parameters, but then I forgot to do that. So yeah, those are useless here

File: db/models.py (hunk 1 above)
Comment (thorium-zero) [reply db/models.py:48]: oh, ok, I didn't realize that, thanks

**Review by YuriiKindrat — APPROVED** [review]

**Pull request closed without merging**